### PR TITLE
Azure ad

### PR DESCRIPTION
**Pull request by vparekh94**
Added AzureAD Example

### CODE REVIEW

File: v0.11/ee-integrating-auth-systems.md [hunk 1]
```diff
@@ -25,6 +25,40 @@ astronomer:
             authUrlParams: # Additional required params set on case-by-case basis
 ```
 
+## AzureAD Example
+To start off, register the application. Note here that the Redirect URI is not optional, it must be:
+
+https://houston.BASEDOMAIN/v1/oauth/direct
```
Comment (petedejoy) [inline v0.11/ee-integrating-auth-systems.md]: @vparekh94 don't think this is right- it should be `/redirect` I believe.

File: v0.11/ee-integrating-auth-systems.md [hunk 2]
```diff
@@ -25,6 +25,40 @@ astronomer:
...
+  houston:
+    config:
+      auth:
+        openidConnect:
+          google:
+            enabled: false
+          microsoft:
+            enabled: true
+            clientId: <client_id>
+            discoveryUrl:<discovery_url>
```
Comment (petedejoy) [inline v0.11/ee-integrating-auth-systems.md]: Add a space after the colon here for clarity.

File: v0.12/ee-installation-eks.md [hunk 3]
```diff
@@ -226,7 +223,7 @@ smtpUrl: smtps://USERNAME:PW@HOST/?pool=true
 If there are `/` or other escape characters in your username or password, you may need to [URL encode](https://www.urlencoder.org/) those characters.
```
Comment (petedejoy) [inline v0.12/ee-installation-eks.md]: Is this an intentional change?

Comment (vparekh94) [reply v0.12/ee-installation-eks.md]: Yes

**Pull request merged**